### PR TITLE
Vdyma/#1524 pepsea improvements

### PR DESCRIPTION
**Pull request by vdyma**
This pull request if accepted will merge PepSeA MSA and MSA panels in one.

### CODE REVIEW

File: packages/Bio/src/package.ts [hunk 1]
```diff
@@ -410,36 +400,65 @@ export async function toAtomicLevel(df: DG.DataFrame, macroMolecule: DG.Column):
...
+  gapOpenInput.setTooltip('Gap opening penalty at group-to-group alignment');
+  const gapExtendInput = ui.floatInput('Gap extend', 0);
+  gapExtendInput.setTooltip('Gap extension penalty to skip the alignment');
+  const inputRootStyles = [methodInput.root.style, gapOpenInput.root.style, gapExtendInput.root.style];
+
+  const colInput = ui.columnInput('Sequence', table, seqCol, () => {
+    const potentialCol = colInput.value;
+    const unUsedName = table.columns.getUnusedName(`msa(${potentialCol.name})`);
+  
+    if (checkInputColumnUi(potentialCol, 'MSA', [NOTATION.FASTA], ['DNA', 'RNA', 'PT'], false)) {
```
Comment (LaykDimon) [inline packages/Bio/src/package.ts:426]: maybe put ['DNA', 'RNA', 'PT'] into const?

File: packages/Bio/src/package.ts [hunk 2]
```diff
@@ -410,36 +400,65 @@ export async function toAtomicLevel(df: DG.DataFrame, macroMolecule: DG.Column):
...
-  return multipleSequenceAlignmentAny(sequence.dataFrame, sequence);
+export function multipleSequenceAlignmentAny(col: DG.Column<string> | null = null): void {
+  const table = col?.dataFrame ?? grok.shell.t;
+  const seqCol = col ?? table.columns.bySemType(DG.SEMTYPE.MACROMOLECULE);
+  if (seqCol == null) {
+    grok.shell.warning(`MSAError: dataset doesn't conain any Macromolecule column`);
+    return;
+  }
+
+  let performAlignment: () => Promise<DG.Column<string> | null> = async () => null;
```
Comment (MariaDolotova) [inline packages/Bio/src/package.ts:413]: why are you using async here? You are not awaiting runKalign/runPepsea

File: packages/Bio/src/utils/pepsea.ts [hunk 3]
```diff
@@ -12,93 +14,62 @@ type PepseaRepsonse = {
...
+    clustersCol = clustersCol.convertTo(DG.TYPE.STRING);
 
   const clusters = clustersCol.categories;
   const bodies: PepseaBodyUnit[][] = new Array(clusters.length);
 
   // Grouping data by clusters
   for (let rowIndex = 0; rowIndex < peptideCount; ++rowIndex) {
     const cluster = clustersCol.get(rowIndex) as string;
-    if (cluster === '')
+    if (cluster == '')
```
Comment (LaykDimon) [inline packages/Bio/src/utils/pepsea.ts:32]: why did you change '=== 'to '=='?

File: packages/Bio/src/package.ts [hunk 4]
```diff
@@ -410,36 +400,65 @@ export async function toAtomicLevel(df: DG.DataFrame, macroMolecule: DG.Column):
...
+        inputRootStyle.display = 'none';
+
+      performAlignment = async () => null;
+    }
+  }) as DG.InputBase<DG.Column<string>>;
+  colInput.setTooltip('Sequences column to use for alignment');
+  colInput.fireChanged();
+
+  let msaCol: DG.Column<string> | null = null;
+  const dialog = ui.dialog('MSA')
```
Comment (MariaDolotova) [inline packages/Bio/src/package.ts:448]: dialog variable is not used anywhere
